src/modules/args.rs

```diff
@@ -8,7 +8,7 @@ pub struct Args {
 	pub address: Option<String>,
 
 	/// Unit of measurement ['c' (°Celsius) | 'f' (°Fahrenheit)]
-	#[clap(short, long, value_parser, default_value_t = String::from("c"))]
+	#[clap(short, long, value_parser, default_value_t = String::new())]
 	pub unit: String,
```

Comment:
To keep a better consistency with the rest of the app I would prefer to use it as an option and remove the default value instead of assigning an empty string to it.

This would require unwrapping the unit [value at line 10 of the params ](https://github.com/tobealive/wthrr-the-weathercrab/blob/5406156a11e70ea69592889ae5d35aa04bd6e084/src/modules/params.rs#L10) as it is done with the address. Would you like to also take over this one?